setup/configure_kernel_ima_module/test.sh

```diff
@@ -24,9 +25,13 @@ rlJournalStart
         # generate key and certificate for IMA
         rlRun "limeInstallIMAKeys"
         # install IMA policy
-        rlRun "limeInstallIMAConfig"
+        rlRun "limeInstallIMAConfig ${IMA_POLICY_FILE}"
         # clear TPM
-        rlRun "tpm2_clear"
+        if ! limeTPMEmulated; then
+            rlRun "tpm2_clear"
+        fi
+        # workaround for issue https://github.com/keylime/keylime/issues/1025
+        rlRun "echo 'd /var/run/keylime 0700 keylime keylime' > /usr/lib/tmpfiles.d/keylime.conf"
```

Comment:
Isn't this expected to be in the packaging?

Reply:
It is. But without a workaround in place I can't run tests successfully and it takes time to get issue 1025 fixed in distribution packages. I plan to remove it once the fix is in place.